WordPress/Classes/Services/CredentialsService.swift

```diff
@@ -10,13 +10,12 @@ struct KeychainCredentialsProvider: CredentialsProvider {
 
 class CredentialsService {
     private let provider: CredentialsProvider
-    private let dotComOAuthKeychainService = "public-api.wordpress.com"
 
     init(provider: CredentialsProvider = KeychainCredentialsProvider()) {
         self.provider = provider
     }
 
     func getOAuthToken(site: JetpackSiteRef) -> String? {
-        return provider.getPassword(username: site.username, service: dotComOAuthKeychainService)
+        return provider.getPassword(username: site.username, service: AppConstants.authKeychainServiceName)
```

Comment:
As I understand, this is the main place of the fix. Using correct `authKeychainServiceName` based on the app, and not a static "public-api.wordpress.com".

Then this `CredentialsService` is used in `ActivityStore` and `PluginStore`. This is the reason Activity log and plugins weren't working.

Reply:
Yes, exactly!